mathics/core/parser/parser.py

```diff
@@ -413,6 +419,18 @@ def p_Decrement(self, token):
         q = prefix_ops['PreDecrement']
         return Node('PreDecrement', self.parse_exp(q))
 
+
+    def p_Definition(self, token):
+        self.consume()
+        q = prefix_ops['Definition']
+        return Node('Definition', self.parse_exp(q))
```

Comment:
I think we can remove this rule now right? It's been replaced with `p_PatternTest`.